README.md

```diff
@@ -46,14 +46,23 @@ We use SASS CSS pre-compiler. If you're doing front-end work your local machine
 ### Installing
     $ git clone https://github.com/uktrade/directory-ui-export-readiness
     $ cd directory-ui-export-readiness
-    $ virtualenv .venv -p python3.5
+    $ virtualenv .venv -p python3.6
     $ source .venv/bin/activate
-    $ pip install -r requirements_test.txt
+    $ python -m  pip install -r requirements_test.txt
 
 ### Running the webserver
     $ source .venv/bin/activate
     $ make debug_webserver
 
+### Running Redis Server
+    $ brew install redis
```

Comment:
Instead of including platform-specific instructions for installing tools we can get away with simply saying something "Install redis on your host"

`brew` assumes mac is being used. Some devs use linux. We don't want the readme to be a general technical manual :)